python/lib/py_wrappers.h

```diff
@@ -83,6 +83,7 @@ void py_settings(py::module& pyopensn);
 
 /// Wrap the solver components of OpenSn (unfinshed).
 void py_solver(py::module& pyopensn);
+void WrapProblem(py::module& slv);
 void WrapSolver(py::module& slv);
 void WrapLBS(py::module& slv);
```

Comment:
for my own comprehension, why are these 3 wrappers having the same arguments? @andrsd @quocdang1998

Reply:
`py::module` is the C++ class that represents a python module that you add the wrapper to. It is not global, so it is passed as an argument. The whole wrapping code is designed around this pattern.
If you are asking why it is `slv`, then my guess is that these APIs are part of the `solver` submodule.